artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/AbstractJournalStorageManager.java

```diff
@@ -1599,7 +1599,7 @@ protected void confirmLargeMessage(final LargeServerMessage largeServerMessage)
             confirmPendingLargeMessage(largeServerMessage.getPendingRecordID());
             largeServerMessage.setPendingRecordID(LargeServerMessage.NO_PENDING_ID);
          } catch (Exception e) {
-            ActiveMQServerLogger.LOGGER.warn(e.getMessage(), e);
```

Comment:
I would just log the warn here.. no need for a special ID!

Reply:
ok